### PR TITLE
Get all albums

### PR DESCRIPTION
**Pull request by 0xAaCE**
## Summary

Add get all albums query with paginated and order

## Trello Card

https://trello.com/c/3vEJQmYT/22-listado-de-albums-paginado-y-ordenado

### CODE REVIEW

File: app/services/album.js [hunk 1]
```diff
@@ -22,3 +22,31 @@ exports.getPhotosOfAlbum = albumId => {
...
+
+exports.getAllAlbums = (offset, limit, orderBy = 'id') => {
+  const endpoint = `${url}albums`;
+  return axios
+    .get(endpoint)
+    .then(response => response.data)
+    .catch(e => {
+      throw errors.conectionError(e.message);
+    })
+    .then(albums => {
```
Comment (gonziesc) [inline app/services/album.js:34]: dont modify your param, create new constants using sort and slice

File: app/services/album.js [hunk 2]
```diff
@@ -22,3 +22,31 @@ exports.getPhotosOfAlbum = albumId => {
...
+      albums.sort((a, b) => {
+        if (!a[orderBy] || !b[orderBy]) {
+          throw errors.badRequest('The orderBy parameter do not exist');
```
Comment (gonziesc) [inline app/services/album.js]: we wont always need the sort param

**Review by gonziesc — CHANGES_REQUESTED** [review]

File: app/services/album.js [hunk 3]
```diff
@@ -22,3 +22,35 @@ exports.getPhotosOfAlbum = albumId => {
...
+            }
+            if (a[orderBy] > b[orderBy]) {
+              return 1;
+            }
+            return 0;
+          })
+          .slice(offset, limit);
+        return slicedAlbums;
+      }
+      const slicedAlbums = albums.slice(offset, limit);
```
Comment (gonziesc) [inline app/services/album.js]: and what if you have both order and limit? can you handle them with this?

**Review by gonziesc — CHANGES_REQUESTED** [review]

File: app/services/album.js (hunk 3 above)
Comment (0xAaCE) [reply app/services/album.js]: Yes, after the sort I add the slice too

**Review by gonziesc — APPROVED** [review]

**Pull request merged**